index.js

```diff
@@ -1,9 +1,11 @@
 'use strict';
+/* eslint no-var: 0 */
 var once = require('once');
+
 var setups;
 
 exports.setups = function (fns) {
-	setups = fns.map(function (fn) {
+	setups = fns.map(function (fn) { /* eslint prefer-arrow-callback: 1*/
```

Comment:
I just want to use es5 xo settings for this file only. I didn't want to list all the rules. but if you think listing all the rules is better, do you think we should move the comments to the top? Thanks.